package.json

```diff
@@ -245,8 +245,8 @@
     "@valora/eslint-config-typescript": "^1.0.2",
     "@valora/resolve-kit": "^1.0.1",
     "@walletconnect/legacy-types": "2.0.0",
-    "@walletconnect/sign-client": "2.10.0",
-    "@walletconnect/types": "2.10.0",
+    "@walletconnect/sign-client": "2.11.3",
+    "@walletconnect/types": "2.11.3",
```

Comment:
do we still need to keep these as exact versions instead of range versions?

Reply:
Fixed in c1c69fffdbe247f1eca6a728b4fad1aa527b6d97!